cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -370,6 +371,11 @@ def filter_to_keys(dict_, keys):
             # Calcs updated from CFE's result
             self.calcs = cfe_calcs
 
+            logger.info("Eligibility result (using CFE): %s", cfe_result)
+            return cfe_result
+        elif self._under_18_passported(self.case_data):
+            self.calcs = cfe_calcs
+
```

Comment:
Think this should be incorporated into the if statement above rather than repeating the cfe call